app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
You probably won't need this. I'd recommending deleting any of the Rails templately files that you aren't using.